examples/jsm/nodes/core/NodeBuilder.d.ts

```diff
@@ -24,6 +24,8 @@ export class NodeBuilder {
 		color: boolean[];
 		lights: boolean;
 		fog: boolean;
+		transparent: boolean;
+		irradiance: boolean;
```

Comment:
Irradiance in NodeBuilder?

Reply:
Yes, it seems to be used here: https://github.com/mrdoob/three.js/blob/dev/examples/jsm/nodes/materials/nodes/StandardNode.js#L229

But there is a big chance I misunderstand something. I don't quite understand this code base and the logic yet.